tests/exemplars.ttl

```diff
@@ -29,6 +29,15 @@ kb:Action-13c91d40-9ca0-4558-81a1-4f7db888c371
 	gufo:hasEndPointInXSDDateTimeStamp "2021-02-03T04:05:06Z"^^xsd:dateTimeStamp ;
 	.
 
+kb:AlternateDataStream-e6ceee40-5614-4740-b08c-d21ce9b2356d
+	a
+		uco-observable:AlternateDataStream ,
```

Comment:
How can something be typed as both a gufo:Category (non-sortal) and a gufo:Kind (sortal). I think that is incorrect.

Reply:
Does this illustration help?

```mermaid
flowchart BT

subgraph TBox
  _uco_obs_ads[uco-observable:AlternateDataStream]
  _uco_obs_fso[uco-observable:FileSystemObject]

  subgraph _gufo_types [gUFO Typing]
    _gufo_category[gufo:Category]
    _gufo_kind[gufo:Kind]
  end
end

subgraph ABox
  _kb_ads_e6[kb:AlternateDataStream-e6ceee40...]
end

_kb_ads_e6 -- ∈ --> _uco_obs_ads
_uco_obs_ads -- ⊂ --> _uco_obs_fso
_uco_obs_ads -- ∈ --> _gufo_kind
_uco_obs_fso -- ∈ --> _gufo_category

_gufo_kind -- ⋂=∅ --> _gufo_category
```

Reply:
`gufo:Category` and `gufo:Kind` are disjoint, via disjointedness amongst their parent classes.

Reply:
Indeed, `gufo:Category` and `gufo:Kind` are disjoint. 
The disjointness doesn't create a conflict for `uco-observable:AlternateDataStream`, since the `gufo:Category` does not provide a principle of identity as `gufo:Kind` does.